NightscoutServiceKit/NightscoutService.swift

```diff
@@ -67,6 +69,11 @@ public final class NightscoutService: Service {
         self.lockedObjectIdCache = Locked(ObjectIdCache())
         self.otpManager = OTPManager(secretStore: KeychainManager())
         self.commandSourceV1 = RemoteCommandSourceV1(otpManager: otpManager)
```

Comment:
REMOTE 2.0: WILL REMOVE PRE-MERGE

(At least the remote 2 initialization lines will be removed)